e2e-tests/tfm/cypress/e2e/journeys/case-amendments/7. is-using-facility-end-date.spec.js

```diff
@@ -0,0 +1,149 @@
+import relative from '../../relativeURL';
+import facilityPage from '../../pages/facilityPage';
+import amendmentsPage from '../../pages/amendments/amendmentsPage';
+import MOCK_DEAL_AIN from '../../../fixtures/deal-AIN';
+import dateConstants from '../../../../../e2e-fixtures/dateConstants';
+import { PIM_USER_1, BANK1_MAKER1, ADMIN } from '../../../../../e2e-fixtures';
+
+context('Amendments - Cover End Date', () => {
+  let dealId;
+  const dealFacilities = [];
+
+  before(() => {
+    cy.insertOneDeal(MOCK_DEAL_AIN, BANK1_MAKER1).then((insertedDeal) => {
+      dealId = insertedDeal._id;
+
+      const { dealType, mockFacilities } = MOCK_DEAL_AIN;
+
+      cy.createFacilities(dealId, [mockFacilities[0]], BANK1_MAKER1).then((createdFacilities) => {
+        dealFacilities.push(...createdFacilities);
+      });
+
+      cy.submitDeal(dealId, dealType, PIM_USER_1);
+    });
+  });
+
+  after(() => {
+    cy.deleteDeals(dealId, ADMIN);
+    dealFacilities.forEach((facility) => {
+      cy.deleteFacility(facility._id, BANK1_MAKER1);
+    });
+  });
+
+  it('should take you to `Has the bank provided a facility end date` page with initially unchecked values', () => {
+    cy.login(PIM_USER_1);
+    const facilityId = dealFacilities[0]._id;
+
+    cy.visit(relative(`/case/${dealId}/facility/${facilityId}`));
+
+    facilityPage.facilityTabAmendments().click();
+    amendmentsPage.addAmendmentButton().should('exist');
+    amendmentsPage.addAmendmentButton().contains('Add an amendment request');
+    amendmentsPage.addAmendmentButton().click();
+    cy.url().should('contain', 'request-date');
+
+    amendmentsPage.amendmentRequestDayInput().clear().focused().type(dateConstants.todayDay);
+    amendmentsPage.amendmentRequestMonthInput().clear().focused().type(dateConstants.todayMonth);
+    amendmentsPage.amendmentRequestYearInput().clear().focused().type(dateConstants.todayYear);
+    amendmentsPage.continueAmendment().click();
+
+    cy.url().should('contain', 'request-approval');
+    amendmentsPage.amendmentRequestApprovalYes().click();
```

Comment:
I'm not a fan of these all being on the same page (they're different pages right?)

Reply:
As in the full journey being on each test? I'm not keen either just following the standard pattern set on the other tests. I am keen though to merge this in without huge refactors, given the e2e testing will change quite a lot in even the next PR I have open (as suddenly it will fork into facility end date/ bank review date). So would prefer to defer any big changes to that one.

Reply:
yep happy to leave, more that all these methods are in a file called `page/amendmentsPage` that actually includes lots of different pages

Reply:
Ah gotcha, I'll take a look as part of the next PR